repository-data/application/src/main/resources/hcm-content/taxonomies.yaml

```diff
@@ -0,0 +1,683 @@
+/content/taxonomies:
```

Comment:
We currently follow `snake_case` (e.g. `aritificial-intelligence-and-robotics`) for keys of value-lists. Could we stick to it for taxonomy category keys as well for consistency ? This way we can retain the dashes given in the spreadsheet (e.g. `covid-19`) to differentiate it from the underscores introduced by us for the category keys. Hope that makes sense ?

Reply:
Have modified accordingly